src/site/content/en/metrics/lcp/index.md

```diff
@@ -381,6 +381,17 @@ performance techniques that can also improve LCP, see:
 * [Optimize your JavaScript](/fast#optimize-your-javascript) (for
   client-rendered sites)
 
+### When and when not to lazy load images
+
+Lazy loading is a common performance optimization in which images and other elements are deferred from loading until they approach the viewport as the user scrolls. It's a time-tested technique that saves bandwidth and improves load performance.
+
+However, lazy loading can affect LCP negatively when used improperly. As a general rule, these is one rule of lazy loading to follow if your LCP candidate is an image: If an LCP candidate is above the fold (or reasonably close), do _not_ lazy load it. The reason for this depends on how you're lazy loading images:
```

Comment:
Also, I wonder if we want to make this advice stronger and more concise. E.g. "Never lazy load any images that are likely to be your LCP element."

Since an LCP candidate needs to be in the viewport in order to be an LCP candidate, I'm not sure we want to talk about "above the fold". My sense is the guidance is two steps:

1. Use devtools/lighthouse to figure out what your LCP element is across a number of common screen sizes
2. Make sure you're not lazy loading any images identified in No. 1